src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private Random random = new Random();
+
+    public Ball getRandomBall() {
+        int index = random.nextInt(100);
+        ColorSupplier supplier = new ColorSupplier();

```

Comment:
```suggestion
    private Random random = new Random();
    private ColorSupplier supplier = new ColorSupplier();

    public Ball getRandomBall() {
        int index = random.nextInt(100);

```
давай ColorSupplier тоже вынесем на уровень класса